frontends/election-manager/src/utils/cast_vote_record_files.ts

```diff
@@ -188,6 +201,77 @@ export class CastVoteRecordFiles {
 
     return result;
   }
+  /**
+   * Parses the CVR files from `files` to determine the number of CVR entries
+   * in the file and the number of those entries that are duplicates.
+   */
+  async parseAllFromFileSystemEntries(
+    files: KioskBrowser.FileSystemEntry[],
+    election: Election
+  ): Promise<CastVoteRecordFileData[]> {
+    const results: CastVoteRecordFileData[] = [];
+    for (const file of files) {
+      const parsedFileInfo = parseCvrFileInfoFromFilename(file.name);
+      try {
+        const importedFile = [...this.files].find((f) => f.name === file.name);
+        if (importedFile) {
+          results.push({
+            fileImported: true,
+            newCvrCount: 0,
+            importedCvrCount: importedFile.importedCvrCount,
+            scannerIds: importedFile.scannerIds,
+            exportTimestamp: importedFile.exportTimestamp,
+            isTestModeResults: this.fileMode === 'test',
+            fileContent: '',
+            name: file.name,
+          });
+        } else {
+          assert(window.kiosk);
+          const fileContent = await window.kiosk.readFile(file.path, 'utf-8');
+          const parsedFile = await this.parseFromFileContent(
+            fileContent,
+            file.name,
+            parsedFileInfo?.timestamp || new Date(file.mtime),
+            parsedFileInfo?.machineId || '',
+            parsedFileInfo?.isTestModeResults ?? false,
+            election
+          );
+          results.push(parsedFile);
+        }
+      } catch (error) {
+        continue;
```

Comment:
I'll add a comment but if there is an error reading the file or it isn't a valid CVR file it might throw. In those cases then its not a valid file to import either so we can just ignore it and not include it in the import wizard.